src/Symfony/Component/Serializer/Encoder/ChainDecoder.php

```diff
@@ -32,6 +32,8 @@ public function __construct(array $decoders = array())
 
     /**
      * {@inheritdoc}
+     * 
+     * @throws RuntimeException If no decoder is found.
```

Comment:
Anything you document an exception as just ``@throws Exception``, it is something you cannot account for as you cannot know what you should be accounting for.
This is why I said that such phpdoc makes sense when documenting specific exceptions for specific error cases which can be accounted for by the calling code (``ContainerInterface::get`` and ``UrlGeneratorInterface::generate`` have such phpdoc)